[2024-10-11T21:50:00.885+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: nba_data_pipeline.create_tables scheduled__2023-10-03T00:00:00+00:00 [queued]>
[2024-10-11T21:50:00.896+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: nba_data_pipeline.create_tables scheduled__2023-10-03T00:00:00+00:00 [queued]>
[2024-10-11T21:50:00.900+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-10-11T21:50:00.903+0000] {taskinstance.py:1280} INFO - Starting attempt 2 of 2
[2024-10-11T21:50:00.907+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-10-11T21:50:00.936+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): create_tables> on 2023-10-03 00:00:00+00:00
[2024-10-11T21:50:00.960+0000] {standard_task_runner.py:55} INFO - Started process 115 to run task
[2024-10-11T21:50:00.993+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'nba_data_pipeline', 'create_tables', 'scheduled__2023-10-03T00:00:00+00:00', '--job-id', '43', '--raw', '--subdir', 'DAGS_FOLDER/nba_data_pipeline.py', '--cfg-path', '/tmp/tmptxxidd8r']
[2024-10-11T21:50:01.000+0000] {standard_task_runner.py:83} INFO - Job 43: Subtask create_tables
[2024-10-11T21:50:01.202+0000] {task_command.py:388} INFO - Running <TaskInstance: nba_data_pipeline.create_tables scheduled__2023-10-03T00:00:00+00:00 [running]> on host eccc01d8c48f
[2024-10-11T21:50:01.478+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=nba_data_pipeline
AIRFLOW_CTX_TASK_ID=create_tables
AIRFLOW_CTX_EXECUTION_DATE=2023-10-03T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-03T00:00:00+00:00
[2024-10-11T21:50:01.487+0000] {logging_mixin.py:137} INFO - Error: could not connect to server: Connection refused
	Is the server running on host "localhost" (127.0.0.1) and accepting
	TCP/IP connections on port 5432?
could not connect to server: Cannot assign requested address
	Is the server running on host "localhost" (::1) and accepting
	TCP/IP connections on port 5432?
[2024-10-11T21:50:01.488+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/scripts/create_tables.py", line 75, in create_tables
    if conn is not None:
UnboundLocalError: local variable 'conn' referenced before assignment
[2024-10-11T21:50:01.509+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=nba_data_pipeline, task_id=create_tables, execution_date=20231003T000000, start_date=20241011T215000, end_date=20241011T215001
[2024-10-11T21:50:01.533+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 43 for task create_tables (local variable 'conn' referenced before assignment; 115)
[2024-10-11T21:50:01.554+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-10-11T21:50:01.599+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
